dir.props

```diff
@@ -125,7 +125,9 @@
 
   <!-- Create a collection of all project.json files for dependency updates. -->
   <ItemGroup>
-    <ProjectJsonFiles Include="$(SourceDir)**/project.json" />
+    <!-- Exclude PackageFiles tool-runtime and test-runtime project.json files. -->
+    <ProjectJsonFiles Include="$(SourceDir)**/project.json"
+                      Exclude="$(SourceDir)Microsoft.DotNet.Build.Tasks\PackageFiles\**\project.json" />
```

Comment:
Well, independently, `Include` uses `/` because what I saw when I added it were a bunch of properties with `/` like https://github.com/dagood/buildtools/blob/exclude-tool-runtime-validation/dir.props#L37-L50. `Exclude` uses `\` because MSBuild expands the `**` with `\`, and paths with `/` won't exclude anything.

I found https://github.com/dotnet/corefx/pull/1310 that explains why there are so many `/` in the build, but I don't know what has happened since then (if we got a better way to transform paths or not). It seems like any path that might find its way to an xplat `Exec` needs to be `/`, but MSBuild-only stuff uses `\`.

I'm open to changing just this instance of `/` to `\` so the different usages at least aren't in the same item.